src/App.js

```diff
@@ -1,12 +1,23 @@
 import React, { Component } from 'react';
```

Comment:
See this step: https://www.carbondesignsystem.com/tutorial/react-step-1#import-carbon-component-styles

Reply:
I imported missing styles